jax/_src/compiler.py

```diff
@@ -304,6 +330,7 @@ def compile_or_get_cached(
       # Host callbacks are currently baked into the HLO module so we cant share
       # them.
       and len(host_callbacks) == 0
+      and _total_graph_constant_size(computation.operation) <= config.share_binary_between_hosts_size_threshold
```

Comment:
Could you add an option to skip this check?
For example if share_binary_between_hosts_size_threshold is <= 0